projects/storefrontapp-e2e-cypress/cypress/integration/regression/my-account/address-book.e2e-spec.ts

```diff
@@ -12,12 +12,12 @@ function addressBookTest() {
     before(() => {
       cy.requireLoggedIn();
       cy.visit('/');
-      cy.get('.cx-nav-link')
+      cy.get('cx-page-layout')
```

Comment:
I'm not a fan, could this not be `cx-login`?

Reply:
Sounds fair, added cx-login to make the targeting more specific